src/features/navigation/RootNavigator/withScreenWrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ComponentType, memo } from 'react'
+import styled from 'styled-components/native'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const withScreenWrapper = (Component: ComponentType<any>, name: string) => {
```

Comment:
est-il possible de supprimer ce disable en typant autrement ?

```suggestion
export const withScreenWrapper = (Component: ComponentType, name: string) => {
```

ou

```suggestion
export const withScreenWrapper = (Component: ComponentType<unknown>, name: string) => {
```

Reply:
(1) 

> est-ce que tu connais les raisons pour lesquelles la lib l'interdit ? 

Je ne sais pas, c'est une convention, j'ai recheck dans la doc mais ils précisent ça en plus des example :  

- The Navigator should contain Screen elements as its children to define the configuration for routes.

> createNativeStackNavigator is a function that returns an object containing 2 properties: Screen and Navigator. Both of them are React components used for configuring the navigator. The Navigator should contain Screen elements as its children to define the configuration for routes.

et surement qu'il y a un passage de props

> Each Stack.Screen component takes a name prop which refers to the name of the route and component prop which specifies the component to render for the route. These are the 2 required props.

(2)

> est-ce que forcer une View (comme le fait cette PR) pourrait causer des problèmes ?

Non, une view flex 1 c'est comme un fragment, je l'avais fait, je sais plus ou, p-e l'erreur boundary, mais je vais par précaution faire un round de test

Reply:
(3)

> est-il possible de supprimer ce disable en typant autrement ?

:heavy_check_mark: